build.sh

```diff
@@ -121,43 +121,44 @@ fi
 # Now we build everything.
 BUILD_DIR="$ROOT_DIR/build/"
 if [ ! -d "${BUILD_DIR}" ]; then
-mkdir -p "${BUILD_DIR}"
+  mkdir -p "${BUILD_DIR}"
 fi
 
 pushd "$BUILD_DIR"
 
 
-need_pickle5_backport=0
-PYTHON_REVISION="$("$PYTHON_EXECUTABLE" -s -c "import sys; print('{}.{}.{}'.format(*sys.version_info))")"
-case "${PYTHON_REVISION}.0" in
-3\.[5-7].*|3.8.[0-1].*) need_pickle5_backport=1;;
-esac
-
-if [ 0 -ne "${need_pickle5_backport}" ]; then
```

Comment:
Ahh @raulchen, you know what... I just saw that you completely got rid of this version check :\ was it deliberate? (Do you expect the behavior to be equivalent?) Or was it an accident? Because I had added it for Windows, and on Python 3.8.2 [we get `pickle5_available=0`](https://github.com/mehrdadn/ray/runs/674986004?check_suite_focus=true#step:5:581) and end up installing pickle5 despite the fact that I thought we don't need it there?

Reply:
Oops. I accidentally introduced this mistake while trying to improve code readability and comment in the last commit. Will fix it right now.

Reply:
Ah okay thanks!